src/xcode/ENA/ENAUITests/UniversalQRCodeScanner/ENAUITests_16_UniversalQRCodeScanner.swift

```diff
@@ -98,5 +98,18 @@ class ENAUITests_16_UniversalQRCodeScanner: CWATestCase {
 		/// Exposure Submission QR Info Screen
 		XCTAssertTrue(app.staticTexts[AccessibilityLabels.localized(AppStrings.ExposureSubmissionQRInfo.title)].waitForExistence(timeout: .short))
 	}
+	
+	func test_TicketValidationTestFromUniversalQRCodeScanner() throws {
+		app.launch()
+		
+		app.buttons[AccessibilityIdentifiers.TabBar.scanner].waitAndTap()
+		
+		/// Simulator only Alert will open where you can choose what the QRScanner should scan
```

Comment:
Do we really need this test? Seems to do the same as `test_isFirstConsentShown` in  `ENAUITests_19_TicketValidation`

If it is still needed I think it would be better to put it in `ENAUITests_19_TicketValidation:`

Reply:
yeah the test is very similar but here we test if the route from the universal scanner is doing correctly and checks only if the correct screen opens. So the test is from my point of view correct in set `ENAUITests_16_UniversalQRCodeScanner`.
The test in `ENAUITests_19_TicketValidation` is more detailed and checks if all elements of the consent screen are existent.
What they have in common is the starting point, because the ticket validation can only be triggered from the universal scanner.
Still need to change @marcussc ?

Reply:
So `ENAUITests_19_TicketValidation` has the same test, but is more detailed? If I understand it correct, I would say just remove it from `ENAUITests_16_UniversalQRCodeScanner `